### PR TITLE
DR tutorial working collaboratively + Share, transfer, delete project

### PR DESCRIPTION
**Pull request by alex-cit**
related to #436 
Dashboard Redesign - screenshots update 
-  https://merginmaps.com/docs/tutorials/working-collaboratively/
- https://merginmaps.com/docs/manage/project-advanced/

### CODE REVIEW

**Review by MarcelGeo — CHANGES_REQUESTED** [review]

File: src/tutorials/working-collaboratively/index.md [hunk 1]
```diff
@@ -7,67 +7,68 @@ In this tutorial you'll learn a few different ways of sharing your project with
...
-   - Set the **Permission in projects**
+4. Now you will see project details. There is a **Collaborators** tab where you can find the list of people who have access to this specific project. 
+
+   To invite a new collaborator, click on the **Share** button.
+   ![Mergin Maps dashboard share project with collaborators](./dashboard-share-project-invite-user.jpg "Mergin Maps dashboard share project with collaborators")
 
-4. Click on **Invite** to send an invitation. 
+5. Enter the email and specify **Project permission**.
+
+   You can enter multiple emails in the **Share with** field and invite multiple people at once.
```
Comment (MarcelGeo) [inline src/tutorials/working-collaboratively/index.md:32]: You add also usernames into Share with

image

**Review by MarcelGeo — CHANGES_REQUESTED** [review]

File: src/manage/project-advanced.md [hunk 2]
```diff
@@ -14,76 +14,64 @@ You can follow our [Working collaboratively](../tutorials/working-collaborativel
...
-If you want to share all projects in your workspace with a group of users, you can invite them to be a **member** of a workspace. It is also possible to invite people as **guests** and share only selected projects with them.
+If you want to share all projects in your workspace with a group of users, you can invite them to become a **member** of a workspace.
 
 You have to be the **admin** or **owner** of the workspace to manage access to the workspace. See [Member Roles and Permissions](./permissions/#workspace-member-roles) for more details.
 
 To add users to a workspace:
 1. Log into <AppDomainNameLink />
-2. Select **Members** from the left panel and click on **Add People**
-   ![Add people in Members tab in Mergin Maps](../tutorials/working-collaboratively/web-workspace-add.jpg "Add people in Members tab in Mergin Maps")
-3. In the invite form, enter the email addresses of people you want to invite.
```
[the remark this answers is not in the document]
Comment (alex-cit) [reply src/manage/project-advanced.md:25]: good points, thanks!

**Comment by alex-cit** [comment]
thanks for the review, it should be fixed now :)

**Review by MarcelGeo — APPROVED** [review]

File: src/manage/project-advanced.md [hunk 3]
```diff
@@ -14,76 +14,67 @@ You can follow our [Working collaboratively](../tutorials/working-collaborativel
...
-   - for **guests**, specify the projects to which they should have access. When inviting guests, you can give them [*read* or *write* permissions](./permissions/#project-permissions) to these projects.
-  ![Invite teammates and set permissions form](./workspace-members-guests.png "Invite teammates and set permissions form")
+2. Go to the **Members** tab in the left panel and click on **Invite**
+   ![Invite people to Mergin Maps workspace](./dashboard-workspace-invite.jpg "Invite people to Mergin Maps workspace")
 
-4. Click on **Invite** to send an invitation. An email with a link will be sent to the recipients. After they accept the invitation, they will become members or guests of your workspace.
+3. In the form, enter the email addresses of people you want to invite and choose their [**workspace role**](./permissions/#workspace-member-roles) from the list.
+   - If you want to grant them access to all projects in the workspace, select one of the member roles (*Reader*, *Writer*, *Admin* or *Owner*).
+   - If you want to invite them only to some projects, select the *Guest* option and specify which projects should be shared with them.
```
Comment (raherin) [inline src/manage/project-advanced.md:28]: Edit text:
If you only want to invite them to some projects, select the *Guest* option and specify which projects should be shared with them.

File: src/manage/project-advanced.md [hunk 4]
```diff
@@ -118,10 +109,10 @@ To avoid synchronisation issues, everyone should delete the old project from all
 
 ### Delete a project through merginmaps.com
 
-1. Log in <AppDomainNameLink /> and choose the project you want to delete
+1. Log in <AppDomainNameLink /> and navigate to the project you want to delete
```
Comment (raherin) [inline src/manage/project-advanced.md:112]: Edit text:
Log in to `<AppDomainNameLink />`

**Pull request merged**